themes/default/content/what-is/what-is-aws-secrets-manager.md

```diff
@@ -234,6 +234,6 @@ class MyStack : Stack
 
 {{% /choosable %}}
 
-- **Advanced secrets management**: Explore Pulumi’s detailed guides on the centralized management of secrets in cloud applications, particularly with Pulumi ESC (Environments, Secrets, and Configurations). For more information, visit the [Pulumi ESC documentation for the AWS Secrets provider](/docs/pulumi-cloud/esc/providers/aws-secrets/).
+- **Advanced secrets management**: For organizations utilizing AWS Secrets Manager alongside other cloud platforms, [Pulumi ESC (Environments, Secrets, and Configurations)](/docs/pulumi-cloud/esc/) offers a centralized solution for managing secrets across multiple environments. This is particularly valuable when your infrastructure spans beyond AWS, allowing for seamless integration and management of secrets from various providers in one place. Moreover, Pulumi ESC supports the generation of dynamic credentials, elevating its utility in scenarios where secrets need to be frequently rotated or updated. Dive deeper into how Pulumi ESC can streamline your secrets management workflows by visiting the Pulumi ESC documentation for the [Pulumi ESC documentation for the AWS Secrets provider](/docs/pulumi-cloud/esc/providers/aws-secrets/).
```

Comment:
isn't alongside other secrets managers or having configs stored elsewhere? 

because as far as i know AWS secrets manager isn't cloud dependent. so its more about if you are using many secrets managers or you also have configs you need to store, ESC can help in this case